_docs_operate/setup-with-docker-compose.md

```diff
@@ -10,16 +10,161 @@ toc: true
 properties:
   - id: SC077
   - category: Connector Setup
-  - description: #2 https//enmeshed.eu/integrate/connector-installation
+  - description: #2 https://enmeshed.eu/integrate/connector-installation
```

Comment:
I would rather not insert any links under 'description', this only happens in about 4 places in the documentation. I would either leave this blank or insert text. (Here and in the Excel)